delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/plugin/PluginStatus.scala

```diff
@@ -0,0 +1,33 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.plugin
+
+import ch.epfl.bluebrain.nexus.delta.sdk.plugin.PluginStatus.Status
+
+/**
+  * Representation of a [[Plugin]] status.
+  *
+  * @param name    name of the [[Plugin]]
+  * @param version version of the [[Plugin]]
+  * @param status  status of the [[Plugin]]
+  */
+final case class PluginStatus(name: String, version: String, status: Status) {}
```

Comment:
the realms PR introduces a new type: Name; I would suggest we use it here instead of a string.